src/core/server/saved_objects/import/resolve_import_errors.test.ts

```diff
@@ -443,6 +444,7 @@ describe('#importSavedObjectsFromStream', () => {
     });
 
     test('handles a mix of successes and errors and injects metadata', async () => {
+      // TODO
```

Comment:
nit: The TODO might be easier to tackle if we give a brief description on what remains to be done in this test.

Reply:
Woups, that's the bookmark I use to keep track of where I am when taking a break 😅 . Will remove it